src/container-service.ts

```diff
@@ -14,7 +14,7 @@ export const ContainerService = {
     input: FileFetcherInput | FileUpdaterInput | ProxyConfig
   ): Promise<void> {
     const tar = pack()
-    tar.entry({name}, JSON.stringify(input))
+    tar.entry({name, mode: 0o777}, JSON.stringify(input))
```

Comment:
Can we name this and move it into a constant or function? I'm always forgetting which permissions are which, so it'd help me out 😇

Reply:
I created one called RWX_ALL, is that clearer?